validator/sawtooth_validator/execution/scheduler.py

```diff
@@ -101,9 +101,18 @@ def finalize(self):
         raise NotImplementedError()
 
     @abstractmethod
-    def complete(self, block):
+    def complete(self, block, force=False):
```

Comment:
Can we name this better?  `force_write_state` perhaps?  Likewise in the other signatures.  `force` by itself doesn't communicate much in the places where it's used.